flow-server/src/main/resources/com/vaadin/flow/server/frontend/tsconfig.json

```
@@ -5,11 +5,13 @@
 // For more information about the configurations, please refer to http://www.typescriptlang.org/docs/handbook/tsconfig-json.html
 {
   "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
     "sourceMap": true,
```

```suggestion
    "sourceMap": true,
```

seems not needed, as `import * as React from 'react';` is preferred